docs/guide/introduction/quickstart.md

```diff
@@ -0,0 +1,77 @@
+# Quickstart
+
+## Initial setup
+
+To start using EasyP, you need to initialize the project.
+
+To do this, run the following command in the terminal:
+
+```bash
+easyp init 
+```
+
+## Setup linter rules
+
+For example, you can use default linter rules.
+
+```yaml
+lint:
+  use:
+    - DEFAULT
+```
+
+After setting up the linter rules, you can run the linter command:
+
+```bash
+easyp lint
+```
+
+## Setup package dependencies
+
+If you ara using third-party packages,
+you need to add them to the `dependencies` section in the `easyp.yaml` file.
+
+```yaml
+lint:
+  use:
+    - DEFAULT
+deps:
+  - github.com/googleapis/googleapis
+  - github.com/grpc-ecosystem/grpc-gateway@v2.20.0
```

Comment:
i'd added:
Format is simple:
$GIT_LINK@$VERSION
where:
$GIT_LINK: is just a link go git repo (github, gitlab etc)
$VERSION: git tag or FULL hash of commit. 
If version is ommited then easyp will download the latest commit from default branch.